src/crm/DynamicsConnections.ts

```diff
@@ -0,0 +1,14 @@
+interface DynamicsConnections {
```

Comment:
Minor - this looks like a singular connection so maybe we could drop the plural in the name?
```suggestion
interface DynamicsConnection {
```